src/examples/completeExample.json

```diff
@@ -0,0 +1,24 @@
+{
+  "nodeType": "formulas",
+  "formulas": [
+    {
+      "value": "~Q",
+      "row": 1,
+      "resolved": false
+    },
+    {
+      "value": " P&D",
+      "row": 2,
+      "resolved": false
+    }
+  ],
+  "forest": [
+    {
+      "nodeType": "contradiction",
+      "formulas": [],
+      "rule": ["X"]
```

Comment:
Is this supposed to be a correct example?
If so, there's a contradiction node closing the tree but no contradictory formulas, so this is incorrect.
If not, maybe change the name to make it clear this is supposed to be an incorrect example.